packages/circuit-ui/components/TierIndicator/TierIndicator.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2025, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { HTMLAttributes } from 'react';
+import { PlusTier } from '@sumup-oss/icons';
+
+import { utilClasses } from '../../styles/utility.js';
+
+export interface TierIndicatorProps extends HTMLAttributes<SVGElement> {
+  /**
+   * The tier name.
+   */
+  variant: 'plus';
+  /**
+   * Choose from 3 sizes.
+   * @default 'm'
+   */
+  size?: 's' | 'm' | 'l';
+}
+const sizeMap: Record<string, '16' | '24' | '32'> = {
+  s: '16',
+  m: '24',
+  l: '32',
+};
+
+export const TierIndicator = ({
+  size = 'm',
+  variant = 'plus',
+  ...props
+}: TierIndicatorProps) => (
+  <>
+    <span className={utilClasses.hideVisually}>{variant}</span>
+    <PlusTier aria-hidden="true" size={sizeMap[size]} {...props} />
+  </>
```

Comment:
It's safer to wrap this inside an actual element rather than a fragment, otherwise this might cause issues in flex or grid layouts where the visually hidden element is still treated as a separate child.

Reply:
@sirineJ Thanks for making the change! Looks like now we have an issue with the element height. The SVG is an line element, so the `div` applies a default line height to it, causing an empty space below the SVG. Setting `line-height: 0` on the `div` should solve this. 

I'd also suggest adding `display: inline-block` to the `div` to match the Badge component.